php/class-wp-cli.php

```diff
@@ -1004,6 +1004,10 @@ public static function error_to_string( $errors ) {
 				return $message;
 			}
 		}
+
+		if ( $errors instanceof \Exception ) {
```

Comment:
We should check for `Throwable` as well, as that is the base interface to catch starting with PHP 7+.

Reply:
Nice catch, I learned something new :)